### PR TITLE
port: add support for Teams get meeting participant API

### LINKED ISSUE #2872: Support for meeting APIs (JS)
See [parent](https://github.com/microsoft/botframework-sdk/issues/5893)

### PR DESCRIPTION
**Pull request by joshgummersall**
Fixes #2872

Also Refactored tests for TeamsInfo class a bit

## Testing
Added unit tests for TeamsInfo method and performed functional testing by sideloading a JS bot into Teams and linking the botbuilder library locally.

### CODE REVIEW

File: libraries/botframework-connector/src/teams/models/mappers.ts [hunk 1]
```diff
@@ -270,12 +270,18 @@ export const TeamsParticipantChannelAccount: msRest.CompositeMapper = {
         className: 'TeamsParticipantChannelAccount',
         modelProperties: {
             ...TeamsChannelData.type.modelProperties,
-            memberRole: {
+            meetingRole: {
                 serializedName: 'memberRole',
```
Comment (EricDahlvang) [inline libraries/botframework-connector/src/teams/models/mappers.ts:274]: should this be 'meetingRole'?

Comment (joshgummersall) [reply libraries/botframework-connector/src/teams/models/mappers.ts:274]: I think I must have fixed this while you were reviewing 😄

**Comment by EricDahlvang** [comment]
Updates to the dotnet sdk:
https://github.com/microsoft/botbuilder-dotnet/pull/4826

**Comment by joshgummersall** [comment]
@EricDahlvang fixed, thanks.

**Review by EricDahlvang — APPROVED** [review]
A little hard to read, with the no-op changes mixed in.  But, i think this looks good.

Thanks @joshgummersall !

**Pull request merged**